README.md

```diff
@@ -129,6 +137,86 @@ Similarly, to put a clojure data structure on the producer channel:
 (>!! producer-chan ["k2" "v2" "events"])
 ```
 
+## Consumer Decorator
+The consumer decorator allows running custom logic on the consumer polling thread.
```

Comment:
If you want each line to be displayed separately, please add two spaces at the end of a line.